README.md

```diff
@@ -0,0 +1,39 @@
+# My Portfolio
+This repository is my portfolio setup that I will be sharing with recruiters to evaluate my past work history and experience. It has been linked to all my social media profile URLs 
+
+![image](https://user-images.githubusercontent.com/78415728/125063659-2d6b7f00-e0c9-11eb-8e7a-0529b700643e.png)
+
+## Built With
+### Major languages
+- HTML
+- CSS
+- Linters
+
+### Technologies used
+- Github
+- Git
+- Visual Studio Code
+- Ubuntu
+
+## Getting Started
+To get a local copy up and running follow these simple example steps.
+git clone git@github.com:faizi2500/hello-world.git
+
+
+## Authors
+👤 Faizan Zahid
+
+GitHub: @faizi2500
+Twitter: @faizi_250
```

Comment:
- [ ] Please use this format for author section `Github: [@faizi2500 ](www.github.com/faizi2500)`

Reply:
Thank you for implementing the previous reviewer change 💯